src/ui/app/components/transactionBuilder.jsx

```diff
@@ -268,24 +341,76 @@ const TransactionBuilder = React.forwardRef(({ onConfirm }, ref) => {
             justifyContent="center"
             flexDirection="column"
           >
-            <Image src={IOHK} width="40px" />
-            <Box h="4" />
             <Text fontSize="sm">
-              Support the development of Nami Wallet by delegating to{' '}
+              Enter the Stake Pool ticker or ID to delegate your funds and start
+              receiving rewards. Alternatively, head to{' '}
               <Link
                 fontWeight="semibold"
-                onClick={() => window.open('https://pipool.online')}
+                onClick={() => window.open('https://pool.pm')}
               >
-                Berry Pool
-              </Link>{' '}
-              and earn approximately <b>3.7%</b> staking rewards per year.
-              Alternatively, head to https://pool.pm/, connect your Nami wallet
-              and delegate to a stake pool of your choice.
+                https://pool.pm
+              </Link>
+              , connect your Nami wallet and delegate to a stake pool of your
+              choice
             </Text>
             <Box h="6" />
-            {data.error ? (
+            <Tooltip
+              label={`${data.pool.ticker} ${data.pool.name}`}
+              placement="top"
+              isOpen={data.pool.state === PoolStates.DONE ? undefined : false}
```

Comment:
when this is set to `true`?

Reply:
Indeed, it is not clear. improved it here: ce24811a58c623edbe48a8a75c471e65cae295f1